### PR TITLE
Adding ambassador edge stack

### PR DESCRIPTION
**Pull request by milindchawre**
Adding Ambassador Edge Stack.

**Screenshots:**

_Default home page_
image

_Ambassador diagnostic page:_
image

_Ambassador edge stack admin page:_
image

### CODE REVIEW

**Comment by milindchawre** [comment]
ping @saiyam1814

File: ambassador/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: ambassador-edge-stack
+title: "Ambassador Edge Stack"
+version: "latest"
```
Comment (saiyam1814) [inline ambassador/manifest.yaml:4]: We are moving away from using the latest version, can we put a stable version for this ?

Comment (milindchawre) [reply ambassador/manifest.yaml]: Done!

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**